pkg/collector/corechecks/oracle-dbm/custom_queries.go

```diff
@@ -166,6 +166,13 @@ func (c *Check) CustomQueries() error {
 							errInQuery = true
 							break
 						}
+					} else if vFloat64, ok := v.(float64); ok {
+						metricRow.value = vFloat64
+						if err != nil {
```

Comment:
where would this error be coming from? is this accidental copy from L156?

Reply:
Yes, thanks! Good catch! Also the error handling for `Int64` was unnecessary. Removed now.